src/PrestaShopBundle/Model/Product/AdminModelAdapter.php

```diff
@@ -589,6 +589,7 @@ private function getCustomFields()
 
         foreach ($customizationFields as $customizationField) {
             $baseObject = [
+                'id_customization_field' => $customizationField[$this->locales[0]['id_lang']]['id_customization_field'],
```

Comment:
Just a though, are you sure this will lead to php warnings ? Because I'm just doing like the other fields, and I'm getting no warning in debug mode